templates/base.html.twig

```diff
@@ -101,8 +101,8 @@
 {% include 'layout/_footer.html.twig' with {inSidebar: true} %}
 {% include 'layout/_topbar.html.twig' %}
 <div id="popover" class="popover js-popover section section--small" role="dialog"></div>
-<div id="scroll-top" data-controller="scroll-top" style="display: none;" data-action="click->scroll-top#scrollTop">
-    <i class="fa-solid fa-arrow-up"></i> <small class="hidden">0</small>
+<div id="scroll-top" data-controller="scroll-top" style="display: none;" data-action="click->scroll-top#scrollTop" aria-hidden="true">
```

Comment:
read a few search results on accessible scroll to top [such as this](https://www.digitala11y.com/scroll-to-top-does-it-need-to-be-accessible/). It seems to go between it not being needed for screenreaders and a general "everything should be accessible". It seems like the goal is to remove non-content, non-navigation from display of screenreaders so I opt'd to hide the scroll to top button from it, but can always change it if that proves incorrect